Apps/WebClient/src/Server/Services/ConfigurationService.cs

```diff
@@ -66,5 +81,21 @@ public MobileConfiguration GetMobileConfiguration()
             this.logger.LogTrace("Getting mobile configuration data...");
             return this.mobileConfig;
         }
+
+        private TourConfiguration? GetTourConfiguration()
+        {
+            TourConfiguration? tourConfig = this.cacheProvider.GetItem<TourConfiguration>(this.tourSettingsCacheKey);
+            if (tourConfig == null)
+            {
+                IList<ApplicationSetting>? settings = this.applicationSettingsDelegate.GetApplicationSettings(TourSettingsMapper.Application, TourSettingsMapper.Component);
+                if (settings.Any())
+                {
+                    tourConfig = TourSettingsMapper.Map(settings);
+                    this.cacheProvider.AddItem(this.tourSettingsCacheKey, tourConfig);
```

Comment:
1. there's a getorset read through cache function that can be used
2. for how long this is going to be cached?

Reply:
@ytqsl , Thanks for the getorset hint, I see its not often used in the code, I'll keep it in mind for future implementations.

There hasn't been any discussion on the expiry for the cache, however we could set it quite high as we would only set this value when we release thus in the sprint cycle. So I would argue once a day should be fine.

Do we clear cache on production deployment? or would we need to target this particular key on deployments?